README-PoC.md

```diff
@@ -0,0 +1,71 @@
+# GraphQL Gateway PoC
+
+## Description
+
+This proof of concept contains Gateway GraphQL server, which communicates via gRPC with Director. Gateway delegates all operations to Director, which contains actual business logic.
+
+From technical point of view, Gateway utilizes connection pool to be easily scalable. gRPC underneath uses HTTP/2 server-side push, so there is no need to reconnect for every request.
+
+## Run locally
```

Comment:
even if we reuse connections, with many requests, provided solution seems to be not the most efficient:
I queried for:
- 100 apps, each has 10 APIs: response time was ~0.04s
- 100 apps without querying about APIs: 0.01s
- 1000 apps, each has 10 APIs: response time ~ 0.5s
- 1000 apps, without querying about APIs: 0.04s

This was tested on the localhost.

Reply:
Although we would go with some pagination, I agree, that's another downsize of this solution.